Sources/StytchUI/StytchB2BUIClient/ViewControllers/SMSOTPEntryViewController/SMSOTPEntryViewModel.swift

```diff
@@ -8,8 +9,29 @@ final class SMSOTPEntryViewModel {
     ) {
         self.state = state
     }
+
+    func smsAuthenticate(code: String) async throws {
+        guard let organizationId = OrganizationManager.organizationId else {
+            throw StytchSDKError.noOrganziationId
+        }
+
+        guard let memberId = MemberManager.memberId else {
+            throw StytchSDKError.noMemberId
+        }
+
+        let parameters = StytchB2BClient.OTP.SMS.AuthenticateParameters(
+            sessionDurationMinutes: state.configuration.sessionDurationMinutes,
+            organizationId: organizationId,
+            memberId: memberId,
+            code: code,
+            setMfaEnrollment: nil
+        )
+        let response = try await StytchB2BClient.otp.sms.authenticate(parameters: parameters)
+        B2BAuthenticationManager.handleSecondaryReponse(b2bAuthenticateResponse: response)
+    }
 }
 
 struct SMSOTPEntryState {
     let configuration: StytchB2BUIClient.Configuration
+    let expirationDate = Date().addingTimeInterval(120)
```

Comment:
Does `addingTimeInterval` default to seconds, or do you need to specify that (somehow)?

Reply:
This is seconds yes, hard coded 2 minutes here, does it need to be configurable?

Reply:
Nope, we don't support configurable OTP expiration in B2B (AFAIK), just checking that it was defaulting to seconds :)